### PR TITLE
Add Epic Connector Scenario for MI

### PR DESCRIPTION
**Pull request by isuruuy429**
This PR includes the Epic Connector scenario for Micro Integrator.

### CODE REVIEW

File: en/docs/emr-connectivity/guides/emr-connector.md [hunk 1]
```diff
@@ -83,4 +83,49 @@ You can further refer to the Epic Documentation on creating an [OAuth 2.0 App](h
     ```
 
 === "Micro Integrator"
-
+    As a Prerequsite [Setup Micro Integrator](../../install-and-setup/setup-mi-server.md)
+
```
Comment (sameeragunarathne) [inline en/docs/emr-connectivity/guides/emr-connector.md:88]: Shall we have a sentence about the overview of the sample scenario we are trying, starting something like below

"The following example demonstrates ....."

File: en/docs/install-and-setup/setup-mi-server.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+
+Go through the following steps to setup the Micro Integrator server for VSCode. 
+
+1. Install WSO2 Micro Integrator Extension in VSCode. 
+
+2. Download the Micro Integrator 4.3.0 distribution as a ZIP file from [here](https://github.com/wso2/micro-integrator/releases/download/v4.3.0/wso2mi-4.3.0.zip).
+3. Extract the zip file and go to wso2mi-4.3.0/conf/deployment.toml
+4. Add the following configurations.
```
Comment (sameeragunarathne) [inline en/docs/install-and-setup/setup-mi-server.md:9]: let's mention the go to deployment toml file here and add the configurations

File: en/docs/install-and-setup/setup-mi-server.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+
+Go through the following steps to setup the Micro Integrator server for VSCode.
```
Comment (sameeragunarathne) [inline en/docs/install-and-setup/setup-mi-server.md:2]: Let's say install Micro integrator server runtime and tooling by pointing to relevant links( runtime : https://mi.docs.wso2.com/en/latest/install-and-setup/install/installing-mi/, tooling: https://mi.docs.wso2.com/en/latest/develop/mi-for-vscode/install-wso2-mi-for-vscode/#install-micro-integrator-for-vs-code)

File: en/docs/emr-connectivity/guides/emr-connector.md [hunk 4]
```diff
@@ -83,4 +83,49 @@ You can further refer to the Epic Documentation on creating an [OAuth 2.0 App](h
...
+                    </epicFhirR4.readById>
+                    <respond/>
+                </inSequence>
+                <faultSequence>
+                </faultSequence>
+            </resource>
+        </api>
+        ```
+    12. Click on 'Build and Run' button in the top left corner. The Micro Integrator server should start running. 
+    13. Clik on 'Try it' and provide the value for the Patient ID.
```
Comment (sameeragunarathne) [inline en/docs/emr-connectivity/guides/emr-connector.md]: Let's give sample curl

Comment (isuruuy429) [reply en/docs/emr-connectivity/guides/emr-connector.md]: Added

File: en/docs/install-and-setup/setup-mi-server.md (hunk 3 above)
Comment (isuruuy429) [reply en/docs/install-and-setup/setup-mi-server.md:2]: Done

File: en/docs/install-and-setup/setup-mi-server.md (hunk 2 above)
Comment (isuruuy429) [reply en/docs/install-and-setup/setup-mi-server.md:9]: Done

File: en/docs/emr-connectivity/guides/emr-connector.md (hunk 1 above)
Comment (isuruuy429) [reply en/docs/emr-connectivity/guides/emr-connector.md:88]: Added

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**